### ISSUE TITLE
who can give me a hand about how to set the paramemters to improve ap on COCO2017?

### ISSUE THREAD

**Issue by kongqiankun** [fizyr/keras-retinanet#819]
hi, friend, I use the code to train on the coco2017 with the backbone Resnet101, and evaluate on the val2017. But the ap is just 34.9. It has a long distance to the paper. So if u can get a better result, please give me some advice. THANKS.

**Comment by hgaiser** [comment]
I think this has been discussed on Slack. Short answer, try different learning rate schedulers.
